docs/SDL/GetPolicyConfigurationData/index.md

```diff
@@ -0,0 +1,97 @@
+## GetPolicyConfigurationData
+
+Type
+: Function
+
+Sender
+: HMI
+
+Purpose
+: To get policy type and property
+
+
+### Request
+
+!!! MUST
+ 
+1. Send GetPolicyConfigurationData request to SDL with: 
+* `policyType` : Policy Struct for which property needs to be read. Must be a [valid policyType struct](https://github.com/smartdevicelink/sdl_core/blob/master/src/components/policy/policy_regular/include/policy/policy_table/types.h#L288)
+* `property`: Used to request a specific property of `policyType`. If omitted, the entire `policyType` json value is returned
```

Comment:
As it is currently defined, this can't be omitted

Reply:
@jacobkeeler  fixed 5c09b11

Reply:
@jacobkeeler, @LuxoftAKutsan please see removed in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/168/commits/741b2a9f12dbb376c97b62b1ae93b52ff3d0cb42